### ISSUE TITLE
feature request: ability to style class-names and id-names differently in HTML source

### ISSUE THREAD

**Issue by chharvey** [atom/language-html#48]
(reposted from https://github.com/atom/atom/issues/5483)

Currently, all attributes are `span.entity.other.attribute-name.html` and their corresponding value is `span.string.quoted.double.html`. This means that all attribute names are styled exactly the same way, and the same is true about attribute values.

Inspired by NetBeans, there should be a way to style **class** names and **id** names differently from other attributes. This can make code more readable and it can make your workflow much faster. This is illustrated below:

![screen shot 2015-02-10 at 12 32 37](https://cloud.githubusercontent.com/assets/1362083/6132507/ef16b28a-b120-11e4-95f6-22699f8018d4.png)

**Comment by postcasio** [comment]
You can already do this for the `id` attribute, at least.

```<div id="some-id">```

Will generate the HTML:

```
<span class="meta tag block any html">
	<span class="punctuation definition tag begin html">&lt;</span>
	<span class="entity name tag block any html">div</span>
	<span class="meta attribute-with-value id html">
		<span class="entity other attribute-name id html">id</span>
		<span class="punctuation separator key-value html">=</span>
		<span class="string quoted double html">
			<span class="punctuation definition string begin html">"</span>
			<span class="meta toc-list id html">some-id</span>
			<span class="punctuation definition string end html">"</span>
		</span>
	</span>
	<span class="punctuation definition tag end html">&gt;</span>
</span>
```

So you can use `.entity.other.attribute-name.id` to match the attribute name, and `.entity.other.attribute-name.id + .punctuation.separator + .string` to match the value.

**Comment by 50Wliu** [comment]
Fixed by #166.
